traveler-swift-core/TravelerKit/Models/Product.swift

```diff
@@ -14,21 +14,18 @@ public protocol Product {
     var id: String { get }
     /// Price
     var price: Price { get }
-    /// Name
+    /// Type
+    var type: ProductType { get }
```

Comment:
avoid using `type`....thats why I had opted for `productType`...`type` is a reserved keyword in almost every language